units/meson.build

```diff
@@ -1,5 +1,7 @@
 # SPDX-License-Identifier: LGPL-2.1+
 
+rlvl = conf.get('HAVE_SYSV_COMPAT') == 1
```

Comment:
please, let's not abbreviate this so unnecessarily. There's nothing wrong with calling this `with_runlevels`.